web/forms.py

```
@@ -106,6 +106,18 @@
 fernet = Fernet(settings.SECURE_MESSAGE_KEY)
 
 
+class TailwindWidgetMixin:
+    """Mixin providing common Tailwind CSS classes for form widgets."""
+    
+    @staticmethod
+    def get_tailwind_attrs():
```

Ruff (0.8.2)

Missing return type annotation for staticmethod `get_tailwind_attrs`

Add return type annotation: `str`

(ANN205)